### ISSUE TITLE
TypeError: Cannot call method 'toString' of undefined

### ISSUE THREAD

**Issue by poulping** [geddy/geddy#88]
Hey All.
I m not sure the problem is not on my end but could not figure out a better place to post.
I have a fresh install of geddy through npm. having tried the first few steps of the README led me to this result:

`[Fri, 24 Feb 2012 21:35:45 GMT] INFO Server starting with config: {
  "environment": "development",
  "workers": 1,
  "port": 4000,
  "debug": true,
  "rotateWorkers": false,
  "rotationWindow": 7200000,
  "rotationTimeout": 300000,
  "logDir": "/path/log",
  "gracefulShutdownTimeout": 30000,
  "heartbeatInterval": 5000,
  "heartbeatWindow": 20000,
  "staticFilePath": "/path/public",
  "sessions": {
    "store": "memory",
    "key": "sid",
    "expiry": 1209600
  },
  "cookieSessionKey": "sdata",
  "metrics": null,
  "detailedErrors": true,
  "hostname": null
}
[Fri, 24 Feb 2012 21:35:45 GMT] INFO Creating 1 worker process.
TypeError: Cannot call method 'toString' of undefined
    at [object Object].createWorker (/usr/local/lib/node_modules/geddy/lib/cluster/master.js:309:20)
    at [object Object].createWorkers (/usr/local/lib/node_modules/geddy/lib/cluster/master.js:299:14)
    at [object Object].<anonymous> (/usr/local/lib/node_modules/geddy/lib/cluster/master.js:220:14)
    at [object Object].runItem (/usr/local/lib/node_modules/geddy/lib/utils/async.js:108:10)
    at [object Object].<anonymous> (/usr/local/lib/node_modules/geddy/lib/utils/async.js:113:12)
    at Array.0 (/usr/local/lib/node_modules/geddy/lib/utils/async.js:148:54)
    at EventEmitter._tickCallback (node.js:188:41)`

so basically it crashes on launch.
Also after creating the app itself i had a deprectaed usage notice:

```
path.existsSync is deprecated. It is now called `fs.existsSync`.
```

Anyone has some clues please ?
